nixos/modules/services/misc/overseerr.nix

```diff
@@ -0,0 +1,62 @@
+{ config, pkgs, lib, ... }:
+
+with lib;
+let
+  cfg = config.services.overseerr;
+in
+{
+  meta.maintainers = [ ];
```

Comment:
If you end up adding yourself as maintainer for the package, it would be cool if you added yourself as maintainer here as well